src/Features/Core/Portable/MoveToNamespace/AbstractMoveToNamespaceService.cs

```diff
@@ -126,8 +142,8 @@ protected AbstractMoveToNamespaceService(IMoveToNamespaceOptionsService moveToNa
         private bool ContainsNamespaceDeclaration(SyntaxNode node)
             => node.DescendantNodes().OfType<TNamespaceDeclarationSyntax>().Any();
 
-        private static bool ContainsMultipleNamespaceInSpine(SyntaxNode node)
-            => node.AncestorsAndSelf().OfType<TNamespaceDeclarationSyntax>().Count() > 1;
+        private static int GetNamespaceInSpineCount(SyntaxNode node)
+            => node.AncestorsAndSelf().OfType<TNamespaceDeclarationSyntax>().Count();
```

Comment:
fwiw, i find this a bit confusing because it has an AncestorsAndSelf call.  Is this ever called on a namepace itself?  if so, it will return itself in the spine count.  not sure if that is desirable/intentional.  this method should actually be doc'ed to indicate what bheavior we expect.  also, if we can limit it to just supporting certain types of nodes (like types only) that wouldbe great.

Reply:
Yea, that's a little confusing indeed, I combined them into one method that count all NS decls on the spine (i.e. from top to bottom)